rpc_proxy.go

```diff
@@ -33,6 +33,15 @@ const (
 	HeaderMacaroon = "Macaroon"
 )
 
+var (
+	// EmptyMacaroonBytes is the byte representation of an empty but
+	// formally valid macaroon.
+	EmptyMacaroonBytes, _ = hex.DecodeString(
```

Comment:
This is just so the `session.IsSuperMacaroon()` function doesn't trip over the dummy data we sent from `litcli` before. That dummy data couldn't be _parsed_ as a macaroon. This dummy macaroon (to which nobody has the root key) will at least be _parseable_, but then will fail any other checks (which is okay, since it also won't be detected as a super macaroon).

Reply:
I updated the commit message to make it a bit more clear.